### ISSUE TITLE
Compilation problems with casacore 3.0.0

### ISSUE THREAD

**Issue by olebole** [casacore/python-casacore#152]
When compiling the 2.2.1 release obn Debian Testing with casacore version 3.0.0, I get the following error:
```
x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -fno-strict-aliasing -g -O2 -fdebug-prefix-map=/build/python-casacore-2.2.1=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -I/usr/include/python2.7 -c src/fitting.cc -o build/temp.linux-amd64-2.7/src/fitting.o
src/fitting.cc: In function 'void init_module__fitting()':
src/fitting.cc:40:3: error: 'casa' has not been declared
   casa::python::register_convert_excp();
   ^~~~
src/fitting.cc:41:3: error: 'casa' has not been declared
   casa::python::register_convert_basicdata();
   ^~~~
src/fitting.cc:42:3: error: 'casa' has not been declared
   casa::python::register_convert_casa_record();
   ^~~~
src/fitting.cc:43:3: error: 'casa' has not been declared
   casa::python::fit();
   ^~~~
error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
```

**Comment by gervandiepen** [comment]
Casacore pre 3.0 used default namespace casa, but in 3.0 the default has
been changed to casacore (to avoid conflicts with the CASA package).
python-casacore has already been changed to use namespace casacore, but I
guess it needs a new release.

**Comment by gijzelaerr** [comment]
this should be solved with the 3.0.0 release https://github.com/casacore/python-casacore/releases/tag/v3.0.0
